internal/pkg/monitor/monitor_integration_test.go

```diff
@@ -47,6 +48,66 @@ func TestSimpleMonitorNonEmptyIndex(t *testing.T) {
 	runSimpleMonitorTest(t, ctx, index, bulker)
 }
 
+func TestSimpleMonitorWithDebounce(t *testing.T) {
+	ctx, cn := context.WithCancel(context.Background())
+	defer cn()
+	ctx = testlog.SetLogger(t).WithContext(ctx)
+
+	index, bulker := ftesting.SetupCleanIndex(ctx, t, ".fleet-actions")
```

Comment:
Shouldn't the test add new docs in `.fleet-policies` index to be more realistic? Policy changes don't have an action document, I think.